fog/ledger/enclave/impl/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fog-ledger-enclave-impl"
-version = "1.1.0"
+version = "1.0.0"
```

Comment:
Don't change this back to 1.0.0, these changes are part of the release process

Reply:
done